corehq/apps/reports/tests/test_esaccessors.py

```diff
@@ -921,6 +921,7 @@ def setUp(self):
         self.reporting = True
         self.case_sharing = False
         self.es = get_es_new()
+        reset_es_index(GROUP_INDEX_INFO)
```

Comment:
Are the calls to `reset_es_index` something that *should* go in `tearDown`, except that too many tests don't clean up ES data for that to be practical?

Reply:
Sort of. Strictly speaking we'd have like "create es index" function here and a "delete es index" function in `tearDown`, but instead we have this function which is "Delete and recreate index" in the `setUp`